gem2rpm/foreman_plugin.spec.erb

```diff
@@ -18,6 +18,7 @@ LICENSES = JSON.load(File.read(File.join(root, 'licenses.json')))
 <%
 has_assets = spec.files.any? { |f| f.start_with?('app/assets/') }
 has_apidoc = spec.files.any? { |f| f =~ %r{\Aapp/controllers/?.*/api/} }
+has_cronjob = spec.files.any? { |f| f =~ %r{\Aextra/.*cron.*} }
```

Comment:
Should we enforce the `.cron` extension? When does it end with something else?

Reply:
Not sure about, as I found `.cron` and `.cron.d`. I can also think someone used `.crontab` and `.cronjob`. But I can restrict it to this list and this should not be to restrictive. What do you think?

Reply:
I'd prefer to standardize on one since we can easily influence/control the sources. I don't have a lot of examples checked out, but a quick check shows `.cron` is used at least in `smart_proxy_omaha` and `smart_proxy_openscap`.

Reply:
I updated it to only use `.cron`.